src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -276,6 +276,30 @@ void SetGlobalPropertiesRequest::on_event(const event_engine::Event& event) {
   }
 }
 
+bool SetGlobalPropertiesRequest::PrepareResultForMobileResponse(
+    ResponseInfo& ui_info, ResponseInfo& tts_info) const {
+  const bool result =
+      CommandRequestImpl::PrepareResultForMobileResponse(ui_info, tts_info);
+  if (ui_info.is_ok && tts_info.is_unsupported_resource) {
+    return true;
+  }
+  return result;
+}
```

Comment:
@VProdanov No. The `ui_info` and `tts_info` are initialized in the `CommandRequestImpl::PrepareResultForMobileResponse` call.